tests/Tests/800__culler/culler.robot

```diff
@@ -147,23 +159,23 @@ Get And Verify Notebook Culler Timeout
 
 Get Notebook Culler Timeout From Configmap
     [Documentation]    Gets the current culler timeout from configmap
-    ${current_timeout} =  OpenShiftLibrary.Oc Get  kind=ConfigMap  name=jupyterhub-cfg
-    ...    namespace=redhat-ods-applications  fields=['data.culler_timeout']
-    ${current_timeout} =  Set Variable  ${current_timeout[0]['data.culler_timeout']}
+    ${current_timeout} =  OpenShiftLibrary.Oc Get  kind=ConfigMap  name=notebook-controller-culler-config
+    ...    namespace=redhat-ods-applications  fields=['data.CULL_IDLE_TIME']
+    ${current_timeout} =  Set Variable  ${current_timeout[0]['data.CULL_IDLE_TIME']}
     [Return]  ${current_timeout}
 
 Get Notebook Culler Timeout From Culler Pod
     [Documentation]    Gets the current culler timeout from culler pod
     ${CULLER_POD} =  Get Notebook Culler Pod Name
-    ${culler_env_timeout} =  Run  oc exec ${CULLER_POD} -c jupyterhub-idle-culler -n redhat-ods-applications -- printenv CULLER_TIMEOUT  # robocop: disable
+    ${culler_env_timeout} =  Run  oc exec ${CULLER_POD} -n redhat-ods-applications -- printenv CULL_IDLE_TIME  # robocop: disable
     [Return]  ${culler_env_timeout}
 
 Modify Notebook Culler Timeout
     [Documentation]    Modifies the culler timeout via UI
     [Arguments]    ${new_timeout}
     Open Dashboard Cluster Settings
     Set Notebook Culler Timeout  ${new_timeout}
-    Sleep  30s  msg=Give time for rollout
+    Sleep  10s  msg=Give time for rollout
 
 Open Dashboard Cluster Settings
```

Comment:
`Open ODS Dashboard With Admin User` and `Verify Cluster Settings Is Available` (defined at 410__ods_dashboard_settings.robot)  do something similar

Reply:
```
[ ERROR ] Error in file '/home/lgiorgi/Work/ods-ci/tests/Tests/400__ods_dashboard/410__ods_dashboard_settings.robot' on line 10: Setting 'Suite Teardown' is not allowed in resource file.
[ ERROR ] Error in file '/home/lgiorgi/Work/ods-ci/tests/Tests/800__culler/culler.robot' on line 8: Error in file '/home/lgiorgi/Work/ods-ci/tests/Tests/400__ods_dashboard/410__ods_dashboard_settings.robot' on line 13: Resource file with 'Test Cases' section is invalid.
```

Cannot use them